pkgs/applications/networking/cluster/kubecfg/default.nix

```diff
@@ -1,22 +1,22 @@
 { lib, buildGoPackage, fetchFromGitHub, ... }:
 
-let version = "0.9.1"; in
+let version = "0.12.0"; in
 
 buildGoPackage {
   name = "kubecfg-${version}";
 
   src = fetchFromGitHub {
-    owner = "ksonnet";
```

Comment:
Why change owner? This is something worth mentioning in the commit message.

Reply:
The `ksonnet` project was discontinued. 
See: https://ksonnet.io/
and
https://github.com/bitnami/kubecfg/issues/247